modules/admin_manual/pages/configuration/files/external_storage/configuration.adoc

```diff
@@ -49,7 +48,7 @@ storage row indicates the storage is ready for use. A red or yellow icon
 indicates that ownCloud could not connect to the external storage, so
 you need to re-check your configuration and network availability.
 
-If there is an error on the storage, it will be marked as unavailable for ten minutes.
+NOTE: If there is an error on the storage, it will be marked as unavailable for ten minutes.
```

Comment:
> ...error on the storage...

what is an "error on the storage"? I guess you mean a connection issue. In case you should rephrase that like:
`If there is an error on the storage...` --> `If there is a connection issue with the target storage...`

Reply:
Please do not mark a comment as resolved until you either have addressed it or you dropped a note why the comment  is resolved.

Reply:
Sometimes github is slower than expected, have been adressed

Reply:
Really strange, I still only see the original changes but not your updates... even if I force reload the browser